src/Parser/Core/Modules/Items/Legion/EyeOfCommand.js

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import { calculateSecondaryStatDefault } from 'common/stats';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import { formatPercentage } from 'common/format';
+
+/**
+ * Eye of Command
+ * Your melee auto attacks increase your Critical Strike by X for 10 sec, stacking up to 10 times. This effect is reset if you auto attack a different target.
+ */
+class EyeOfCommand extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+  eyeOfCommand = {
```

Comment:
Why is this an object allowing for multiple spells? Can't there only be one buff as a result of this item?

Reply:
Oh woops - fixing.